### PR TITLE
Igblast parser

### PR DESCRIPTION
**Pull request by aerijman**
Add igblast-parser
----

A simple tool to parse igblast results into a csv format, which is easy to consume and ready for analysis.

### CODE REVIEW

File: recipes/igblast-parser/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  url: "https://github.com/aerijman/{{ name }}/releases/download/v0.0.3/igblast_parser-{{ version }}.tar.gz"
+  sha256: c688be522cb9a887dca8baf4a07cc6805ccacda0d639a9743ba59c497c838f56
+
+build:
+  number: 0
+  noarch: python
+  script: $PYTHON setup.py install --single-version-externally-managed --record=record.txt 
+
+requirements:
```
Comment (rpetit3) [inline recipes/igblast-parser/meta.yaml:18]: I think you will need to add, a build requirement to bring gcc in for you.

```
  build:
    - {{ compiler('cxx') }}
```

Comment (aerijman) [reply recipes/igblast-parser/meta.yaml:18]: Thank you very much for the help!

Why cxx?

Comment (aerijman) [reply recipes/igblast-parser/meta.yaml:17]: I didn't know you were changing the yaml. I kept yours. Just pushed to activate circleci. I thought that I might have mess with the check. I hope will be fine!

Comment (aerijman) [reply recipes/igblast-parser/meta.yaml:17]: I did not include the .c file to allow for users with or without cython. I will add module1.c file.

Comment (aerijman) [reply recipes/igblast-parser/meta.yaml:17]: mmmm... Just noticed that module1.c **IS** in there...

Comment (rpetit3) [reply recipes/igblast-parser/meta.yaml:17]: are there any C++ files or just this C file?

Comment (aerijman) [reply recipes/igblast-parser/meta.yaml:17]: just the c

Comment (aerijman) [reply recipes/igblast-parser/meta.yaml:17]: In the tar.gz I don't see the `.c` file only the `.pyx`.
It's generated from cython.

Should I add manually the `.c` file in the `.tar.gz`?

**Comment by rpetit3** [comment]
@aerijman success!

I'd like someone else to review the recipe to make sure everything looks good

**Comment by rpetit3** [comment]
@bioconda-bot please add label

**Comment by aerijman** [comment]
@rpetit3 Thanks a lot!!

**Comment by davebx** [comment]
@BiocondaBot please fetch artifacts

**Comment by davebx** [comment]
@BiocondaBot please merge

Thanks, @aerijman and @rpetit3

**Pull request merged**